package.json

```diff
@@ -118,5 +118,41 @@
   },
   "insights": {
     "appname": "rbac"
+  },
```

Comment:
This was intentionally removed. I guess a rebase has failed. The routes config moved to a JS file to remove the webpack warnings.